src/clj/harja/kyselyt/laskut.sql

```diff
@@ -12,6 +12,59 @@ WHERE l.urakka = :urakka
   AND l.erapaiva BETWEEN :alkupvm ::DATE AND :loppupvm ::DATE
   AND l.poistettu IS NOT TRUE;
 
+-- name: hae-urakan-johto-ja-hallintokorvaus-raporttiin-aikavalilla
+select tr.nimi as "nimi",
+       tr.id as "tehtavaryhma",
+       tr.jarjestys as "jarjestys",
+       jhk."toimenkuva-id",
+       sum(jhk.tuntipalkka * jhk.tunnit) as "summa"
+from johto_ja_hallintokorvaus jhk
+  join tehtavaryhma tr on tr.yksiloiva_tunniste = 'a6614475-1950-4a61-82c6-fda0fd19bb54'
+where jhk."urakka-id" = :urakka
```

Comment:
tää magic number tai magic string voisi ansaita kommentin, mikä se on miksi näin. kyseessä muistaakseni johto- ja hallintokorvauksen spesifinen ja pysyvä tunniste, mutta tällaiset asiat näyttää uudelle devaajalle todella oudoilta sitten 2v päästä, siksi kommentti olis hyvä